### PR TITLE
docs: Add ionicons api documentation

### PR DESCRIPTION
**Pull request by NotMyFault**

### CODE REVIEW

File: src/main/webapp/Symbols/symbol-ionicons-reference.jelly [hunk 1]
```diff
@@ -0,0 +1 @@
+<l:icon src="symbol-symbolName-outline plugin-ionicons-api-plugin" />
```
Comment (timja) [inline src/main/webapp/Symbols/symbol-ionicons-reference.jelly:1]:
```suggestion
<l:icon src="symbol-symbolName-outline plugin-ionicons-api" />
```

File: src/main/resources/io/jenkins/plugins/designlibrary/Symbols/index.jelly [hunk 2]
```diff
@@ -92,11 +92,29 @@
...
+        ${%View the complete list of symbols provided by Jenkins on GitHub}
         <l:icon src="symbol-external plugin-design-library" />
       </a>
     </span>
 
+    <h2 class="jdl-heading">${%ioniconsApi}</h2>
+    <p class="jdl-paragraph">${%customSymbols.ionicons-api}</p>
+
+    <span>
+      <a href="https://plugins.jenkins.io/ionicons-api/#dependencies" target="_blank" class="jdl-paragraph jdl-link">
```
Comment (timja) [inline src/main/resources/io/jenkins/plugins/designlibrary/Symbols/index.jelly:104]: Page is not visible yet and we should hold off publishing this till present

**Review by timja — APPROVED** [review]
Looks good in general, the text on that page is overly bold for me in Chrome and it doesn't match the text on other pages (e.g. spacing or table).
Not caused by this PR though.

2 issues that should be addressed but approving in general

File: src/main/resources/io/jenkins/plugins/designlibrary/Symbols/index.jelly (hunk 2 above)
Comment (NotMyFault) [reply src/main/resources/io/jenkins/plugins/designlibrary/Symbols/index.jelly:104]: Yeah 👍🏻 

How often does the update center job run? The initial release was ~16h ago, but https://updates.jenkins-ci.org/current/update-center.json doesn't offer it yet.

Comment (timja) [reply src/main/resources/io/jenkins/plugins/designlibrary/Symbols/index.jelly:104]: should be done, I would raise a helpdesk ticket

**Pull request merged**